### PR TITLE
QGM Readme: Add Mac instructions for visualizing the graphs.

### PR DESCRIPTION
**Pull request by wangandi**
The instructions for rendering the graphs in the QGM tests did not work on my Mac laptop, so I have added instructions for Mac.

### CODE REVIEW

File: test/sqllogictest/chbench.slt [hunk 1]
```diff
@@ -1613,3 +1613,27 @@ ORDER BY substr(c_state, 1, 1)
 Finish order_by=(#0 asc) limit=none offset=0 project=(#0..#2)
 
 EOF
+
+query T multiline
```
Comment (asenac) [inline test/sqllogictest/chbench.slt:1617]: How is this change related? :)

Comment (wangandi) [reply test/sqllogictest/chbench.slt:1617]: Oops. In my attempt to rebase this branch on an earlier version of main (since there's that build error), I appear to have accidentally checked in test code from another branch.

**Review by asenac — APPROVED** [review]
